homeassistant/components/sensor/__init__.py

```diff
@@ -88,6 +88,14 @@
 class SensorDeviceClass(StrEnum):
     """Device class for sensors."""
 
+    ACCUMULATED_PRECIPITATION = "accumulated_precipitation"
+    """Accumulared precipitation.
+
+    Unit of measurement: `UnitOfAccumulatedVolumtericFlux` enum
+    - SI /metric: `mm`
+    - USCS / imperial: `in`
```

Comment:
Formatting discrepancy.
```suggestion
    - SI / metric: `mm`
    - USCS / imperial: `in`
```

Reply:
Resolved